metadata/config.yaml

```diff
@@ -40,34 +40,60 @@ DENEB_FORK_EPOCH: 0
 ELECTRA_FORK_VERSION: 0x60000910
 ELECTRA_FORK_EPOCH: 2048
 
+# Fulu
+FULU_FORK_VERSION: 0x70000910
+FULU_FORK_EPOCH: 50688
+
 # Time parameters
 # ---------------------------------------------------------------
-# 12 seconds
+# 12 seconds (*deprecated*)
 SECONDS_PER_SLOT: 12
+# 12000 milliseconds
+SLOT_DURATION_MS: 12000
 # 14 (estimate from Eth1 mainnet)
-SECONDS_PER_ETH1_BLOCK: 12
-# 2**8 (= 256) epochs ~27 hours
+SECONDS_PER_ETH1_BLOCK: 14
+# 2**8 (= 256) epochs
 MIN_VALIDATOR_WITHDRAWABILITY_DELAY: 256
-# 2**8 (= 256) epochs ~27 hours
+# 2**8 (= 256) epochs
 SHARD_COMMITTEE_PERIOD: 256
-# 2**11 (= 2,048) Eth1 blocks ~8 hours
+# 2**11 (= 2,048) Eth1 blocks
 ETH1_FOLLOW_DISTANCE: 2048
+# 1667 basis points, ~17% of SLOT_DURATION_MS
+PROPOSER_REORG_CUTOFF_BPS: 1667
+# 3333 basis points, ~33% of SLOT_DURATION_MS
+ATTESTATION_DUE_BPS: 3333
+# 6667 basis points, ~67% of SLOT_DURATION_MS
+AGGREGATE_DUE_BPS: 6667
+
+# Altair
+# 3333 basis points, ~33% of SLOT_DURATION_MS
+SYNC_MESSAGE_DUE_BPS: 3333
+# 6667 basis points, ~67% of SLOT_DURATION_MS
+CONTRIBUTION_DUE_BPS: 6667
 
 # Validator cycle
 # ---------------------------------------------------------------
 # 2**2 (= 4)
 INACTIVITY_SCORE_BIAS: 4
 # 2**4 (= 16)
 INACTIVITY_SCORE_RECOVERY_RATE: 16
-# 2**4 * 10**9 (= 16,000,000,000) Gwei
-EJECTION_BALANCE: 16000000000
-# 2**2 (= 4)
+# 28,000,000,000 Gwei to ensure quicker ejection
+EJECTION_BALANCE: 28000000000
```

Comment:
This change also seems unrelated to Fusaka? At least I don't remember it being discussed.

(It probably can't even be changed like this for a live network, right?)